Formula/s/sf.rb

```diff
@@ -0,0 +1,25 @@
+class Sf < Formula
+  desc "Command-line toolkit for Salesforce development"
+  homepage "https://developer.salesforce.com/tools/salesforcecli"
+  url "https://registry.npmjs.org/@salesforce/cli/-/cli-2.61.8.tgz"
+  sha256 "6d7388565f83c15edd2ad05d1f42737de64680b8ea18a7f569435e93e00d6620"
+
+  license "BSD-3-Clause"
+
+  depends_on "node@20"
+  # Depends on v20 to avoid the `punycode` warnings
```

Comment:
we can probably remove this
```suggestion
```

Reply:
Leaving it for the moment, ideally we would be removing the dependency on a specific version.

Reply:
if so, can you log an upstream issue for tracking such thing (these warnings do not make sense for us, and they are also just warnings. )